plentymarketsPOS.adoc

```diff
@@ -443,26 +445,66 @@ Die Daten einer Stornierung mit der Herkunft POS können im ERP-System plentymar
 
 ===== Buchungsablauf bei Retouren
 
-Über plentymarkets POS erstellte Retouren werden im ERP-System plentymarkets als Auftrag des Typs *Retoure* ohne Hauptauftrag erstellt. Das bedeutet, dass datentechnisch keine Verbindung zwischen dem ursprünglichen Auftrag und der Retoure besteht.
+Retouren können im Kassensystem plentymarkets POS auf zwei Arten erstellt werden:
+
+* Als Auftrag des Typs *Retoure*, der dem Hauptauftrag zugeordnet wird (ab plentymarkets POS Version 1.8)
+* Als Auftrag des Typs *Retoure* ohne Hauptauftrag
+
+Bis plentymarkets POS Version 1.7.x wurden über plentymarkets POS erstellte Retouren im ERP-System plentymarkets als Auftrag des Typs *Retoure* ohne Hauptauftrag erstellt. Das bedeutet, dass datentechnisch keine Verbindung zwischen dem ursprünglichen Auftrag und der Retoure besteht. Dieses Vorgehen ist auch weiterhin möglich, wenn z.B. der Hauptauftrag nicht bekannt ist. Ab Version 1.8 kann der Hauptauftrag im Kassensystem aufgerufen werden. Sowohl vollständige Retouren als auch eine Teilretoure der Auftragspositionen ist möglich. Verkaufsgutscheine können jedoch nicht retourniert werden.
 
-Die Vorgehensweise zum Retournieren von Artikeln wird im Handbuch des ERP-Systems plentymarkets beschrieben:  +
+Die Vorgehensweise zum Retournieren von Artikeln mit und ohne Hauptauftrag wird im Handbuch des ERP-Systems plentymarkets beschrieben:  +
 https://knowledge.plentymarkets.com/omni-channel/pos/pos-kassenbenutzer#175
 
-Folgende Daten werden für Retouren systemseitig gespeichert und auf dem Beleg dokumentiert:
+Folgende Daten werden für Retouren auf dem Beleg dokumentiert:
 
 * Datum und Uhrzeit der Retoure
 * Art des Geschäftsvorfalls
 * Belegnummer
 * Retournierte Artikel
 * Summe der Retoure
-* Mehrwertsteuersätze
+* Mehrwertsteueranteil
 * Brutto- und Nettobetrag der Retoure
+* Belegnummer des Hauptauftrags (nur bei Zuordnung)
 * Ersteller/-in
+
+Folgende Daten werden für Retouren im ERP-System plentymarkets gespeichert:
+
+* Datum und Uhrzeit der Retoure
+* Art des Geschäftsvorfalls
+* Belegnummer
 * ID der Retoure
+* Retournierte Artikel
+* Summe der Retoure
+* Mehrwertsteueranteil
+* Brutto- und Nettobetrag der Retoure
+* Auftrags-ID und Belegnummer des Hauptauftrags (nur bei Zuordnung)
```

Comment:
Hier auch nur die Auftrags-ID des Hauptauftrages wird an der Retoure gespeichert. Um die Belegnummer des Hauptauftrages zu bekommen muss man in dem Hauptauftrag schauen